pkg/server/test/write_assertions.go

```diff
@@ -89,13 +96,17 @@ func TestWriteAssertions(t *testing.T, datastore storage.OpenFGADatastore) {
 		t.Run(test._name, func(t *testing.T) {
 			model := githubModelReq
 
-			modelID, err := commands.NewWriteAuthorizationModelCommand(datastore, logger, 256*1_024).Execute(ctx, model)
+			writeAuthzModelCmd := commands.NewWriteAuthorizationModelCommand(
```

Comment:
Can we follow up and replace this kind of stuff (here and elsewhere) with the options builder pattern? The default should be used if the option isn't explicitly set. You shouldn't have to explicitly set it if it's a default 😄

Reply:
e.g.

```go
NewWriteAuthorizationModelCommand(datastore, WithLogger(logger), WithAuthorizationModelSizeInBytes(2048)) // if non-defaults are desired, otherwise

NewWriteAuthorizationModelCommand(datastore)
```

Reply:
https://github.com/openfga/openfga/issues/1039